### PR TITLE
Task 3 - get categories: Model, Controller, App and Tests

### PR DESCRIPTION
**Pull request by mgrindrod92**
Submitted alongside general setup, as Tasks 1 and 2 were done on 'Main' by mistake, although this was overseen by a tutor on Zoom who said I could carry on as I had done the work correctly.

(Not sure which link to send in this case)

https://github.com/mgrindrod92/Backend-Project-NC-Games-.git
https://github.com/mgrindrod92/Backend-Project-NC-Games-/compare/3-GET/api/categories?expand=1

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    return db.end()
+});
+
+describe('GET api/categories', () => {
+    it('200: returns a list of categories of games with the correct format', () => {
+        return request(app).get('/api/categories').expect(200).then(res => {
+            expect(res.body.categories.length).toBe(4);
+            expect(Array.isArray(res.body.categories)).toBe(true);
+
+            console.log(res.body.categories);
```
Comment (dappernerddesigns) [inline __tests__/app.test.js:30]: Make sure to remove console.logs that you don't need anymore.

File: app.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+const express = require('express');
+const { getCategories } = require('./controller/category_controller');
+
+const app = express();
+app.use(express.json());
```
Comment (dappernerddesigns) [inline app.js:5]: Do you need this for a get request?

**Review by dappernerddesigns — APPROVED** [review]
Just a few comments to consider, please make sure your commit messages are informative about what the commit does. Take out your console log and app.use(express.json()) <---only add this when you need it. This is still get requests.

File: __tests__/app.test.js (hunk 1 above)
Comment (mgrindrod92) [reply __tests__/app.test.js:30]: Removed now. Thanks.

**Pull request merged**